src/index.html

```diff
@@ -17,11 +17,139 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo Moyo"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div id="product-container"></div>
+    </main>
+
+    <script>
+      const products = [
+        {
+          imgSrc: 'images/imac.jpeg',
+          altText: 'Apple iMac',
+          title: 'APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)',
+          code: '195434',
+          reviews: 5,
+          price: '$2,199',
+        },
+      ];
+
+      const productContainer = document.getElementById('product-container');
+
+      products.forEach((product) => {
+        const card = document.createElement('div');
+        card.className = 'card';
+        card.setAttribute('data-qa', 'card');
+
+        card.innerHTML = `
+          <img class="card__img" src="${product.imgSrc}" alt="${product.altText}" />
```

Comment:
Ensure that `product.imgSrc` is a valid URL or consider adding error handling or a default image in case the image fails to load.